src/util/Constants.js

```diff
@@ -112,6 +112,17 @@ function makeImageUrl(root, { format = 'webp', size } = {}) {
   if (size && !AllowedImageSizes.includes(size)) throw new RangeError('IMAGE_SIZE', size);
   return `${root}.${format}${size ? `?size=${size}` : ''}`;
 }
+
+function makeStickerUrl(root, { format = 'png', size, passthrough = true } = {}) {
+  const params = new URLSearchParams();
+  if (size) {
+    if (!AllowedImageSizes.includes(size)) throw new RangeError('IMAGE_SIZE', size);
+    params.append('size', size);
+  }
+  params.append('passthrough', passthrough);
+  return `${root}.${format}${params}`;
```

Comment:
```suggestion
  return `${root}.${format}?${params}`;
```

currently this doesn't include a `?` before the params, resulting in an invalid URL such as `https://cdn.discordapp.com/stickers/748293547572330607/067a41f0df36a1d0925078ded077824c.pngpassthrough=true`